libs/lizard-connector/lizard_connector/queries.py

```diff
@@ -0,0 +1,204 @@
+        north_east (list, tuple): coordinates of the most north eastern
+                                  point of the bounding box of the form
+                                  (lat, lon).
+
+    Returns:
+        bounding box polygon in Well-known text (WKT).
+    """
+    min_lat, min_lon = south_west
+    max_lat, max_lon = north_east
+    polygon_coordinates = [
+        [min_lon, min_lat],
+        [min_lon, max_lat],
+        [max_lon, max_lat],
+        [max_lon, min_lat],
+        [min_lon, min_lat],
+    ]
+    return wkt_polygon(polygon_coordinates)
+
+
+def in_bbox(south_west, north_east, endpoint=None):
+    """
+    Find all locations within a certain bounding box.
+
+    returns records within bounding box using Bounding Box format (min Lon,
+    min Lat, max Lon, max Lat). Also returns features with overlapping
+    geometry.
+
+    Args:
+        endpoint (str): endpoint name as it is available in the Lizard-api
+        south_west (list, tuple): coordinates of the most south western
+                                  point of the bounding box of the form
+                                  (lat, lon).
+        north_east (list, tuple): coordinates of the most north eastern
+                                  point of the bounding box of the form
+                                  (lat, lon).
+
+    Returns:
+        a query dictionary.
+    """
+    query = "geom_within" if endpoint == 'timeseries' else "in_bbox"
+    return QueryDictionary({query: bbox(south_west, north_east)})
+
+
+def distance_to_point(distance, lat, lon):
+    """
+    Query for records with distance meters from a given point.
+
+    Distance in meters is converted to WGS84 degrees and thus an approximation.
+
+    Args:
+        distance (int): meters from point
+        lon (int): longtitude of point
+        lat (int): latitude of point
+
+    Returns:
+        a query dictionary.
+    """
+    coords = commaify(lon, lat)
+    return QueryDictionary(distance=distance, point=coords)
+
+
+def datetime_limits(start=None, end=None):
+    """
+    Query for date time limits (start, end).
+
+    Args:
+        start (datetime.datetime): start date time.
+        end (datetime.datetime): end date time.
+
+    Returns:
+        a query dictionary.
+    """
+    if not end:
+        end = datetime.datetime.now()
+    start = jsdatetime.datetime_to_js(start) if isinstance(
+        start, datetime.datetime) else start
+    end = jsdatetime.datetime_to_js(end) if isinstance(
+        end, datetime.datetime) else end
+    return QueryDictionary(start=start, end=end)
+
+
+def organisation(organisation_id=None, endpoint=None):
+    query_key = {
+        None: "",
+        "organisation": "",
+        "location": "organisation__"
+    }.get(endpoint, "location__organisation__") + "unique_id"
+
+    org_query = QueryDictionary()
+    if isinstance(organisation_id, str):
+        org_query.update({query_key: organisation_id})
```

Comment:
if ``organisation_id`` is an unicode string ``isinstance(x, str)`` will yield False. Beter use ``isinstance(x, basestring)``